test/libyul/evmCodeTransform/eof_swap256.yul

```diff
@@ -0,0 +1,2837 @@
+//     /* "":8757:8774   */
+//   swapn{212}
+//     /* "":8770:8773   */
+//   0xeb
+//     /* "":8757:8774   */
+//   calldataload
+//     /* "":8795:8812   */
+//   swapn{214}
+//     /* "":8808:8811   */
+//   0xec
+//     /* "":8795:8812   */
+//   calldataload
+//     /* "":8833:8850   */
+//   swapn{216}
+//     /* "":8846:8849   */
+//   0xed
+//     /* "":8833:8850   */
+//   calldataload
+//     /* "":8871:8888   */
+//   swapn{218}
+//     /* "":8884:8887   */
+//   0xee
+//     /* "":8871:8888   */
+//   calldataload
+//     /* "":8909:8926   */
+//   swapn{220}
+//     /* "":8922:8925   */
+//   0xef
+//     /* "":8909:8926   */
+//   calldataload
+//     /* "":8947:8964   */
+//   swapn{222}
+//     /* "":8960:8963   */
+//   0xf0
+//     /* "":8947:8964   */
+//   calldataload
+//     /* "":8985:9002   */
+//   swapn{224}
+//     /* "":8998:9001   */
+//   0xf1
+//     /* "":8985:9002   */
+//   calldataload
+//     /* "":9023:9040   */
+//   swapn{226}
+//     /* "":9036:9039   */
+//   0xf2
+//     /* "":9023:9040   */
+//   calldataload
+//     /* "":9061:9078   */
+//   swapn{228}
+//     /* "":9074:9077   */
+//   0xf3
+//     /* "":9061:9078   */
+//   calldataload
+//     /* "":9099:9116   */
+//   swapn{230}
+//     /* "":9112:9115   */
+//   0xf4
+//     /* "":9099:9116   */
+//   calldataload
+//     /* "":9137:9154   */
+//   swapn{232}
+//     /* "":9150:9153   */
+//   0xf5
+//     /* "":9137:9154   */
+//   calldataload
+//     /* "":9175:9192   */
+//   swapn{234}
+//     /* "":9188:9191   */
+//   0xf6
+//     /* "":9175:9192   */
+//   calldataload
+//     /* "":9213:9230   */
+//   swapn{236}
+//     /* "":9226:9229   */
+//   0xf7
+//     /* "":9213:9230   */
+//   calldataload
+//     /* "":9251:9268   */
+//   swapn{238}
+//     /* "":9264:9267   */
+//   0xf8
+//     /* "":9251:9268   */
+//   calldataload
+//     /* "":9289:9306   */
+//   swapn{240}
+//     /* "":9302:9305   */
+//   0xf9
+//     /* "":9289:9306   */
+//   calldataload
+//     /* "":9327:9344   */
+//   swapn{242}
+//     /* "":9340:9343   */
+//   0xfa
+//     /* "":9327:9344   */
+//   calldataload
+//     /* "":9365:9382   */
+//   swapn{244}
+//     /* "":9378:9381   */
+//   0xfb
+//     /* "":9365:9382   */
+//   calldataload
+//     /* "":9403:9420   */
+//   swapn{246}
+//     /* "":9416:9419   */
+//   0xfc
+//     /* "":9403:9420   */
+//   calldataload
+//     /* "":9441:9458   */
+//   swapn{248}
+//     /* "":9454:9457   */
+//   0xfd
+//     /* "":9441:9458   */
+//   calldataload
+//     /* "":9479:9496   */
+//   swapn{250}
+//     /* "":9492:9495   */
+//   0xfe
+//     /* "":9479:9496   */
+//   calldataload
+//     /* "":9517:9534   */
+//   swapn{252}
+//     /* "":9530:9533   */
+//   0xff
+//     /* "":9517:9534   */
+//   calldataload
+//     /* "":9555:9572   */
+//   swapn{254}
+//     /* "":9568:9571   */
+//   0x0100
+//     /* "":9555:9572   */
+//   calldataload
+//     /* "":9581:9594   */
+//   swapn{256}
```

Comment:
Not sure how valuable this test case really is... but here we have ``swapn{256}``... generating one more variable in the test case, correctly triggers a swap-too-deep.
Also note that we don't run any optimizer here. If we rematerialized here, the code would even become entirely swap-free (and we'd need to prevent reordering by introducing side-effects - luckily without the optimizer this remains rather straight-forward).

Reply:
IMO better to have it than not. Sometimes unexpected things happen at the limits.

Reply:
BTW, we could make these at least *a little* less verbose by adding a test setting for disabling the source locations. I.e. the equivalent of `--debug-info none`. I'd keep them on by default, but often they don't add much and it would be nice to be able to silence them.

Maybe then we'd also get the functional formatting (i.e. `calldatalad(0x0100)`) to make it even more compact - not sure when that kicks in, but I suspect it may be the comments preventing it here.